cmake/modules/CheckCompiler.cmake

```diff
@@ -249,11 +246,3 @@ check_cxx_source_compiles(
 #endif
 int main() {}
 " GLIBCXX_USE_CXX11_ABI)
-
-#---Print the final compiler flags--------------------------------------------------------------------
```

Comment:
Why is this removed?

Reply:
It's partially redundant with what is already printed in the top-level `CMakeLists.txt.

However, it was not *completely* redundant. I updated the other message print to include all the information printed here.